scalalib/src/mill/scalalib/JvmWorkerModule.scala

```diff
@@ -51,6 +53,11 @@ trait JvmWorkerModule extends OfflineSupportModule with CoursierModule {
 
   def zincLogDebug: T[Boolean] = Task.Input(Task.ctx().log.debugEnabled)
 
+  def useShortJvmPath(jvmId: String): Boolean =
+    Properties.isWin && (
+      jvmId.startsWith("graalvm") || jvmId.startsWith("liberica-nik")
```

Comment:
What's `liberica-nik`?

Reply:
It's an alternative GraalVM distribution. Coursier uses it, as it has support for more platforms (like Alpine Linux on arm64, but not Windows on arm64 though)